app/controllers/wikis_controller.rb

```diff
@@ -1,6 +1,8 @@
 class WikisController < ApplicationController
+    before_action :authorize_user, except: [:index, :new, :create]
+
   def index
-    @wikis = Wiki.visible_to(current_user)
+    @wikis = policy_scope(Wiki)
```

Comment:
I can make comments here and I see you listed as a collaborator.  Have I not set this up correctly?